doc/classes/ImageTexture.xml

```diff
@@ -23,6 +23,7 @@
 		var image : Image = texture.get_image()
 		[/codeblock]
 		An [ImageTexture] is not meant to be operated from within the editor interface directly, and is mostly useful for rendering images on screen dynamically via code. If you need to generate images procedurally from within the editor, consider saving and importing images as custom texture resources implementing a new [EditorImportPlugin].
+		[b]Note:[/b] Do not create [ImageTexture] using [code]ImageTexture.new()[/code], as it's not possible to set the image afterwards. Always use [method create_from_image] to create new image texture.
```

Comment:
Can't you use the newly bound `set_image` to set the data after `new()`?

Reply:
ah right forgot about this part